src/components/reusuables/cards/ContentCard.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import { StaticImage } from "gatsby-plugin-image"
+const ContentCard = (props) => {
```

Comment:
Change props to the prop name. Example, (props)-> ({heading,subtitle})

Reply:
Props changed to prop name